Presentation.Web/Controllers/API/DataProcessingRegistrationController.cs

```diff
@@ -453,6 +458,56 @@ public HttpResponseMessage RemoveInsecureThirdCountry(int id, [FromBody] SingleV
                 .Match(_ => Ok(), FromOperationError);
         }
 
+        [HttpGet]
+        [Route("{id}/data-processing-registration-options")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public HttpResponseMessage GetDataProcessingRegistrationOptions(int id, [FromUri] string nameQuery = null, [FromUri] int pageSize = 25)
```

Comment:
Kig din parameterliste igennem Det giver ingen mening. Samtidig skal vi ikke kræve ID til dette endpoint. Det er et endpoint så vi kan få udfaldsrum ud for modulet og kan resolve dem i root scope og genbruge dem i både overblik og alle efterfølgende undersider. Parameterlisten skal kun tage et organizationId for det er dét der er nøglen til udfaldsrummenes tilgængelighed.